docs/BitcoinCoreKeys.md

```diff
@@ -0,0 +1,119 @@
+is less secure than for Eclair (because it is shared among several services for example).
+
+## Configuring Eclair and Bitcoin Core to use a new Eclair-backed bitcoin wallet
+
+Follow these steps to delegate onchain key management to eclair:
+
+1) Generate a BIP39 mnemonic code and passphrase
+
+You can use any BIP39-compatible tool, including most hardware wallets.
+
+2) Create an `eclair-signer.conf` configuration file add it to eclair's data directory
+
+A signer configuration file uses the HOCON format that we already use for `eclair.conf` and must include the following options:
+
+ key                      | description                                                              
+--------------------------|--------------------------------------------------------------------------
+ eclair.signer.wallet     | wallet name                                                              
+ eclair.signer.mnemonics  | BIP39 mnemonic words                                                     
+ eclair.signer.passphrase | passphrase                                                               
+ eclair.signer.timestamp  | wallet creation UNIX timestamp. Set to the current time for new wallets. 
+
+This is an example of `eclair-signer.conf` configuration file:
+
+```hocon
+{
+   eclair {
+      signer {
+         wallet = "eclair"
+         mnemonics = "legal winner thank year wave sausage worth useful legal winner thank year wave sausage worth useful legal winner thank year wave sausage worth title"
+         passphrase = ""
+         timestamp = 1686055705
+      }
+   }
+}
+```
+
+3) Configure Eclair to handle private keys for this wallet
+
+Set `eclair.bitcoind.wallet` to the name of the wallet just created (`eclair` in the example above) and restart Eclair.
+Eclair will automatically create a new, empty, descriptor-enabled, watch-only wallet in Bitcoin Core and import its descriptors.
+
+:warning: Eclair will not import descriptors if the timestamp set in your `eclair-signer.conf` is more than 2 hours old. If the mnmeonics and
+passphrase that your are using are new, you can safely update this timestamp, but if they have been used before you will need to follow
+the steps described in the next section.
+
+You now have a Bitcoin Core watch-only wallet for which only your Eclair node can sign transactions. This Bitcoin Core wallet can
+safely be copied to another Bitcoin Core node to monitor your onchain funds.
+
+You can also use `eclair-cli getmasterxpub` to get a BIP32 extended public key that you can import into any compatible Bitcoin wallet
+to create a watch-only wallet (Electrum for example) that you can use to monitor your Bitcoin Core balance.
+
+:warning: this means that your Bitcoin Core wallet cannot send funds on its on (since it cannot access private keys to sign transactions).
```

Comment:
typo: on its on -> on its own

Reply:
fixed in https://github.com/ACINQ/eclair/pull/2613/commits/c00ae33f62d53d4d3710f7dc05a273993eb47657